src/application_config/app_config.h

```diff
@@ -1,8 +1,37 @@
+/**
+ * @file app_config.h
+ * @author Otso Jousimaa <otso@ojousima.net>
+ * @date 2020-04-16
+ * @brief Master configuration base
+ *       Variables which can be adjusted by application_mode_xxxxx.h are preceeded by #ifdef
+ *       2021-12-10 DG12 correct _FLASH_PAGES from 16 to 0x15000/4096-1 see loader configuration.
+ *       2021-11-22 DG12 add comments and rearrange bringing most likely overided items to the top.
+ *       2021-10-01 Otso add dataformats for encrypted  08, FA
+ *
+ * Types of configuration items:
```

Comment:
app_config has configurable constants, such as flash record IDs. They should be in one place to make it easier to spot possible conflicts

Reply:
Constants that are NOT configurable should be in a separate file.